src/frontend/src/pages/AdminToolsPage/AdminToolsBOMConfig.tsx

```diff
@@ -6,7 +6,7 @@ import { Box } from '@mui/system';
 
 const AdminToolsBOMConfig: React.FC = () => {
   return (
-    <Box>
+    <Box paddingBottom="4">
```

Comment:
The padding at the bottom isn't working 
![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/124754518/66f84b9e-d126-4270-b458-d3eec26a3680)
Look at the syntax it should be 

paddingBottom={4}